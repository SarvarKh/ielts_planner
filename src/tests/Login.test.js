import React from 'react';
import renderer from 'react-test-renderer';
import Login from '../components/presentation/Login';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly', () => {
  const tree = renderer.create(
  <BrowserRouter>
    <Login />
  </BrowserRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
