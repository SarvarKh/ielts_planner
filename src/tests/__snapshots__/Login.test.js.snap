// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="authen-page"
>
  <div>
    <i
      className="fas fa-user-circle"
    />
  </div>
  <form
    onSubmit={[Function]}
  >
    <input
      placeholder="Your email"
      required={true}
      type="email"
    />
    <input
      placeholder="Password"
      required={true}
      type="password"
    />
    <button
      type="submit"
    >
      Log in
    </button>
  </form>
  <div
    className="authen-text"
  >
    <div>
      Have not you signed up yet?
    </div>
    <div>
      Go to the
       
      <a
        href="/sign_up"
        onClick={[Function]}
      >
        Sign up page
      </a>
    </div>
  </div>
  <div
    className="ielts-text authen-text"
  >
    <div>
      Check out
       
      <a
        href="https://www.ielts.org/about-ielts/what-is-ielts"
        onClick={[Function]}
        target="_blank"
      >
        the link
      </a>
       
      to find out more about IELTS.
    </div>
  </div>
</div>
`;
